src/pages/Galllery/comingsoon.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export function ComingSoon() {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "center",
```

Comment:
This is not the recommended way to put styles. I think you should use modular CSS files.